src/main/scala/fr/polytechnique/cmap/cnam/etl/sources/Sources.scala

```diff
@@ -64,7 +69,7 @@ object Sources {
     mcoPath: Option[String] = None,
     mcoCePath: Option[String] = None,
     hadPath: Option[String] = None,
-    ssrPath: Option[String] = None,
+    ssrPaths: Option[List[String]] = None,
```

Comment:
Why passing a list of paths here?

Reply:
As I tried to explain, one line in the SSR_C corresponds to one hospitalStay BUT one line in the SSR_B corresponds to one week of of care. So we add to split SSR into SSR_C and SSR_SEJ (SSR_B, SSR) and we do the join here). It is not costly because SSR does not have a lot of data.

Here is our corresponding conf file for the flattening: 
```
{
    name = "SSR_SEJ"
    join_keys = ["ETA_NUM", "RHA_NUM", "RHS_NUM"]
    main_table_name = "SSR_B"
    tables_to_join = ["SSR_CCAM", "SSR_D", "SSR_CSARR"]//, "SSR_MED", "SSR_MEDATU", "SSR_FH"] TODO: "SSR_C"
  }
```

Reply:
Why is the join for? Does any Extractor need an information both in SSR_SEJ and SSR?

Reply:
In fact, every extractor need the data that are in SSR_C and SSR_SEJ.

That is why we create SSR = (SSR_C joined SSR_SEJ). 
The extractors only see SSR.

The best methodology would be to make this join in SCALPEL-Flattening but  it would need to add some code:
- first make the SSR_SEJ by joining [SSR_B, SSR_CCAM, SSR_D, SSR_CSARR] with (ETA_NUM, RHA_NUM, RHS_NUM) 
- then make the SSR by joining [SSR_SEJ, SSR_C] with (ETA_NUM, RHA_NUM)

Reply:
I see. That seems a bit far fetch for the Flattening part. Can you please add a @TODO flag saying this has to be removed in future versions?

Reply:
You mean moved from featuring to flattening ? I am not sure where we will put this problematic join in future versions ..

Reply:
In the flattening. The join is not a responsibility of the Extraction part.

Reply:
todo added